.github/workflows/e2e-ci.yaml

```diff
@@ -501,5 +518,22 @@ jobs:
       with:
         name: e2e-test-results
         path: ${{ env.TESTS_RESULTS_FOLDER }}/E2E/
+  
+  power_off_azure_vm:
+    name: Power OFF EFLOW
```

Comment:
you confirm this call is idempotent right? If the vm is already deallocated this won't fail?

Reply:
![YesYepGIF](https://user-images.githubusercontent.com/1822123/147105918-feee1d70-c588-43a0-8c54-a2a167e5a101.gif)